add-cognito-custom-attributes.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+
+#
+# This is a temporary script used to add the custom attribute `userId` to our cognito
+# user pools without deleting and recreating them. 
```

Comment:
I want to confirm that I understand why this script is necessary - does Terraform destroy/recreate user pools when adding a custom attribute?

Reply:
Yes, it does! Please let us know if we can answer anything else about this script!